src/client/cli/cmd/launch.h

```diff
@@ -46,13 +46,16 @@ class Launch final : public Command
 private:
     ParseCode parse_args(ArgParser* parser);
     ReturnCode request_launch(const ArgParser* parser);
-    ReturnCode mount_home(const ArgParser* parser);
+    ReturnCode mount(const ArgParser* parser, QString mount_source, QString mount_target);
     bool ask_bridge_permission(multipass::LaunchReply& reply);
 
     LaunchRequest request;
     QString petenv_name;
     std::unique_ptr<multipass::AnimatedSpinner> spinner;
     std::unique_ptr<multipass::utils::Timer> timer;
+
+    std::map<QString, QString> mount_routes;
```

Comment:
The only reason was that I thought ordering would seem more consistent to the user, since the directories would be mounted in the order they were specified. But otherwise it is indeed not necessary

Reply:
Hmm, that is a valid point.  I think about it some.

Reply:
Ok, I think I'm convinced by your reason here.  If someone wanted to nest mounts, say something like `--mount foo:test1 --mount bar:test1/test2`, then the order matters.  So let's leave the `std::map` here for that reason.